bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -933,8 +933,9 @@ Logger getLogger() {
     /**
      * Retrieves an instantiated plugin of the given type, or null if none has been created
      */
+    @SuppressWarnings("rawtypes")
     @Nullable
-    Plugin getPlugin(@NonNull Class<Plugin> clz) {
+    Plugin getPlugin(@NonNull Class clz) {
```

Comment:
Is this required? Worth noting on the PR changeset if so.

Reply:
Updated changeset description